### PR TITLE
Packaging in the Python standard way

### PR DESCRIPTION
**Pull request by dahlia**
I would like it to be submitted on PyPI so that we can install it using `pip install typeannotations`, so I worked on its packaging.

First of all, I wrote a setup.py script for the package, and it consists of several metadata about the package.  `version` is one of them, and I fill it with arbitrary version: 1.0.0.  Of course it can be adjusted by editing setup.py.  There is also `long_description`, which is shown for PyPI package page.  It should be reStructuredText rather than Markdown in order to be rendered by PyPI, so I reformatted README into reStructuredText.  Fortunately GitHub and PyPI both render it well.

To submit the package to PyPI, you first need to register its name.  This process is only required once:

``` console
$ python3 setup.py register
```

After it’s registered, then build the package and upload it:

``` console
$ python3 setup.py sdist upload
```

For more explanation, see [Python Packaging User Guide](https://packaging.python.org/en/latest/tutorial.html#uploading-your-project-to-pypi) as well.

Thanks!

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    description='A library with a set of tools for annotating types '
+                'in Python code.',
+    long_description=readme(),
+    url='https://github.com/ceronman/typeannotations',
+    author='Manuel Cerón',
+    author_email='ceronman' '@' 'gmail.com',
+    license='Apache License',
+    packages=['annotation'],
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
```
Comment (ceronman) [inline setup.py:27]: I don't think this library is Production/Stable so better downgrade this a bit.

**Comment by ceronman** [comment]
Appart from the Development Status thing, this looks good for me. Thanks for sending the pull request! I'm going to merge it and fix the status in a new commit.

**Comment by dahlia** [comment]
Thanks for merging!

**Comment by dahlia** [comment]
@ceronman Could you submit the package to PyPI?

**Pull request merged**